docs/learn/learn-accounts.md

```diff
@@ -112,13 +111,36 @@ For instance, `//bill//account//1` and `//john/polkadot/initial` are both valid.
 derived account, you must know both the seed and the derivation path, so you should either use a
 well-defined sequence (e.g. //0, //1, //2...) or be sure to write down any derivation paths you use.
 
-
-:::info 
+:::info
 
 It is not possible to generate a derived account without also knowing the derivation path.
 
 :::
 
+There is an additional type of derivation called password derivation. On Polkadot you can derive a
+**password key** account using **///** after the mnemonic phrase
+
+```
+'caution juice atom organ advance problem want pledge someone senior holiday very///0'
+```
+
+In this type of derivation, if the seed would leak, accounts cannot be derived without the initial
```

Comment:
by "seed", do you mean the “mnemonic phrase" displayed above?

Reply:
Are seed phrase and mnemonic phrase the same?

Reply:
Yeah, I think so.
You refer to the string of words as mnemonic phrase above and refer to it as seed below. So, wanted to ensure consistency.